articles/logic-apps/quickstart-create-first-logic-app-workflow.md

```diff
@@ -157,7 +157,7 @@ For example, here is a sample email that this logic app sends.
 
 ![Sample email sent when new RSS feed item appears](./media/quickstart-create-first-logic-app-workflow/monitor-rss-feed-email.png)
 
-Technically, when the trigger checks the RSS feed and finds new items, the trigger fires, and the Logic Apps engine creates an instance of your logic app workflow that runs the actions in the workflow. If the trigger doesn't find new items, the trigger doesn't fire and "skips" instantiating the workflow.
+Technically, when the trigger checks the RSS feed and finds new items, the trigger fires, and the logic apps engine creates an instance of your logic app workflow that runs the actions in the workflow. If the trigger doesn't find new items, the trigger doesn't fire and "skips" instantiating the workflow.
```

Comment:
Actually, this instance refers to the service name, which is capitalized, so please change to "Azure Logic Apps". Thanks!